src/internal/connector/sharepoint/list.go

```diff
@@ -18,13 +18,14 @@ import (
 // be found at: https://learn.microsoft.com/en-us/graph/api/resources/list?view=graph-rest-1.0
 // Note additional calls are required for the relationships that exist outside of the object properties.
 
-// loadLists is a utility function to populate the List object.
+// loadSiteLists is a utility function to populate a collection of SharePoint.List
+// objects associated with a given siteID.
 // @param siteID the M365 ID that represents the SharePoint Site
 // Makes additional calls to retrieve the following relationships:
 // - Columns
 // - ContentTypes
 // - List Items
-func loadLists(
+func loadSiteLists(
```

Comment:
This doesn't strike me as useful rename.  We're only handling site lists at the moment, and the func is already housed in the sharepoint package.

Reply:
Leaving the refactor for clarity to developers. It is true that we do not save personnel sites within `gc.SiteIDs`. This naming emphasizes that this call will only return lists that are strictly associated with a site. During prototyping, lists were added to several pages by users. Corso queries identify these lists as the possession of the user and not the site.